test/unit/reductions/exponentiated_gradient/test_exponentiatedgradient_smoke.py

```diff
@@ -69,17 +69,46 @@ def setup_method(self, method):
                         "best_gap": 0.000000, "last_t": 5,
                         "best_t": 5, "disp": 0.005000,
                         "error": 0.442883, "n_oracle_calls": 19,
-                        "n_classifiers": 6}]
+                        "n_classifiers": 6},
+                       {"cons_class": ErrorRateRatio, "eps": 0.1,
+                        "best_gap": 1.1102230246251565e-16, "last_t": 5,
```

Comment:
shouldn't the gap reach 0? I see that it's super close, just wondering

Reply:
let's get back to this after `gamma` is fixed.

Reply:
It checks for precision till 6 decimal places in tests. Fixed this to 0.000000
Fixed `gamma` too.